tsl/src/compression/deltadelta.c

```diff
@@ -736,7 +736,7 @@ zig_zag_encode(uint64 value)
 	/* since shift is underspecified, we use (value < 0 ? 0xFFFFFFFFFFFFFFFFull : 0)
 	 * which compiles to the correct asm, and is well defined
 	 */
-	return (value << 1) ^ (((int64) value) < 0 ? 0xFFFFFFFFFFFFFFFFull : 0);
+	return (value << 1) ^ (((int64) value) < 0 ? 0xFFFFFFFFFFFFFFFFULL : 0);
```

Comment:
This is an example of why I think that uppercase suffixes are very hard to read and lowercase suffixes are preferred. If this the the default for clang-tidy we should, however, use that rather than invent our own rules.

Reply:
CERT C recommends upper case L to avoid confusion with 1, so I guess all casings are ugly in their own way :)